### PR TITLE
ROX-22668: Cluster init bundle operator content changes

### PR DESCRIPTION
**Pull request by alwayshooin**
image

### CODE REVIEW

**Review by kcarmichael08 — APPROVED** [review]
LGTM! You asked about having `<stackrox>` and I think it's ok to not have it in brackets. We want them to use stackrox, although they technically could use anything.

**Review by vjwilson — APPROVED** [review]
I defer to Kerry, who has approved this content, but if you need a dev to approve the code changes ;)--then here is that approval, too.

**Comment by alwayshooin** [comment]
Merging as the failing e2e tests have nothing to do w this PR

**Pull request merged**
